test/jdk/java/lang/instrument/VerifyLocalVariableTableOnRetransformTest.java

```diff
@@ -120,8 +137,92 @@ public VerifyLocalVariableTableOnRetransformTest(String name)
         // Table (LVT) attribute so the class file bytes seen by the
         // retransformClasses() call will not match the class file bytes
         // seen at initial class load time.
-        assertTrue(fTargetClassName + " did not match .class file",
-            fTargetClassMatches);
+        compareClassFileBytes(false);
+    }
+
+    private File originalTargetClassFile() {
+        String resourceName = fTargetClassName + ".class";
+        return new File(System.getProperty("test.classes", "."), resourceName);
+    }
+
+    private File transformClassFile() {
+        // This file will get created in the test execution
+        // directory so there is no conflict with the file
+        // in the test classes directory.
+        String resourceName = fTargetClassName + ".class";
```

Comment:
This name can be defined out of methods `originalTargetClassFile` and `transformClassFile`. The method name `transformClassFile` is confusing. I'd suggest to rename it to `transformedClassFile` or `modifiedClassFile`. Also, the name `originalTargetClassFile` can be shortened to `originalClassFile`.

Reply:
done